index.bs

```diff
@@ -3060,7 +3061,7 @@ a [=moment=] |triggerTime|, and a [=boolean=] |fenced|:
     1. Set |triggerContextID| to |value|["<code>[=trigger-registration JSON key/trigger_context_id=]</code>"].
 1. Let |attributionScopes| be null.
 1. If |value|["<code>[=trigger-registration JSON key/attribution_scopes=]</code>"] [=map/exists=]:
-    1. Set |attributionScopes| to the result of running [=parse attribution scopes=] with |value| and [=max number of attribution scopes=].
+    1. Set |attributionScopes| to the result of running [=parse attribution scopes for trigger=] with |value| and [=max number of attribution scopes=].
 1. If |attributionScopes| is an error or [=set/is empty|empty=], return null.
```

Comment:
Now that we have a separate function for parsing scopes for triggers, can we move the empty check to that algorithm?